roles/jd-client/Cargo.toml

```diff
@@ -29,4 +29,4 @@ tracing-subscriber = { version = "0.3" }
 error_handling = { version = "0.1", path = "../../utils/error-handling" }
 nohash-hasher = "0.2.0"
 key-utils = { version = "^1.0.0", path = "../../utils/key-utils" }
-secp256k1 = { version = "0.27.0", default-features = false, features =["bitcoin_hashes","alloc","rand","rand-std"] }
+secp256k1 = { version = "0.28.2", default-features = false, features =["alloc","rand","rand-std"] }
```

Comment:
Maybe we should put it in the stratum-common crate and import secp256k1 everywhere from there?

Reply:
I was thinking also that using secp256k1 in the roles is not the best thing. This logic should stay in roles-logic-sv2 or key-utils. But I guess that this will be better suited for a new PR.